src/pdsfile/pdscache.py

```
@@ -457,25 +460,23 @@ def wait_and_block(self, funcname=''):
                 return was_blocked
 
             self.logger.warn(f'Process {self.pid} was outraced by {test_pid} ' +
-                             f'while waiting to block')
+                             'while waiting to block')
 
     def unblock(self, flush=True):
```

Ruff (0.16.0)

[warning] Boolean default positional argument in function definition

(FBT002)